submissions/tavor118sn/task_js_post_oop/index.js

```diff
@@ -0,0 +1,90 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+
+class Creature {
+  constructor(species, name, age, gender, legs, say, friends) {
+    this.species = species;
+    this.name = name;
+    this.age = age;
+    this.gender = gender;
+    this.legs = legs;
+    this.saying = say;
+    this._friends = friends
+  }
+
+  get friends() {
+    let friendsRepresentation = '';
+    if (this._friends && this._friends.length > 0) {
```

Comment:
```suggestion
    if (this._friends && this._friends.length) {
```
You can do check like this